src/main/java/nl/tudelft/contextproject/webinterface/WebServer.java

```diff
@@ -73,56 +68,37 @@ public int getUniqueClientCount() {
 	 * 		if an exception occurs while attempting to start the server
 	 */
 	public synchronized void start(int port) throws Exception {
-		//Check if the server is already running.
 		if (running) throw new IllegalStateException("Server is already running");
-		
-		//Generator QRcode on server start
+
 		QRGenerator.getInstance().generateQRcode();
-		
-		//Mark the server as now running and set the port
+
 		this.running = true;
 		this.port = port;
-		
-		//We create the server
+
 		server = new Server(this.port);
 		
 		//We need a way to track users, so we use a session manager that uses cookies.
-		
-		//Create the session id manager
 		sessionIdManager = new HashSessionIdManager();
 		server.setSessionIdManager(sessionIdManager);
-		
-		//Create the session manager
+
 		HashSessionManager sessionManager = new HashSessionManager();
 		sessionManager.setSessionTrackingModes(EnumSet.of(SessionTrackingMode.COOKIE));
 
-		//Set the cookie config
 		SessionCookieConfig scc = sessionManager.getSessionCookieConfig();
 		scc.setComment("CoC Session Cookie");
 		scc.setName("COC_SESSION");
 		
 		SessionHandler sessionHandler = new SessionHandler(sessionManager);
-		
-		//Create the handler that chains everything together.
+
 		contextHandler = new ServletContextHandler(ServletContextHandler.SESSIONS);
-		
-		//Handle requests on /
 		contextHandler.setContextPath("/");
-		
-		//Fetch pages from the webinterface folder
 		contextHandler.setResourceBase(FileUtil.getFile("/webinterface/").getAbsolutePath());
-
-		//Set the session handler
 		contextHandler.setSessionHandler(sessionHandler);
-		
-		//Add the handler to the server
 		server.setHandler(contextHandler);
-		
-		//Add a servlet for handling sessions
+
 		ClientServlet cs = new ClientServlet(this);
 		contextHandler.addServlet(new ServletHolder(cs), "/");
-		
-		//Start the webserver
+
 		server.start();
```

Comment:
Sometimes, I think obvious comments are okay. Removing all the comments from this code makes it more difficult to read, and harder to understand in my opinion. I think that every separate section (separated by the newlines) should have a comment above it here.